### PR TITLE
Start of Sidebar Structure

### PR DESCRIPTION
**Pull request by thunderBiscuit**
This PR modifies the `config` file in order to create a sidebar that is more structured. You'll note that it removes the headers being expanded for each selected file.

I also used the toml version of the config file because I think when those get big the json format is super hard to parse for the eye, but I also added the json and yaml format for the config files. All three produce the same result. I'm happy to use any of them, I just wanted to point out that we can use a different one if we want.

This will allow us to compare apples to apples a bit more when looking at GitBook.

Cheers!

### CODE REVIEW

**Comment by MaxHillebrand** [comment]
This sounds good!!
But I'm a bit too lazy [and not even sure how to do it properly] to compile this branch of a fork of a repo? :laughing: 

Could you please save me some trouble and post a screen shot?

**Comment by dennisreimann** [comment]
@thunderBiscuit Why is the config present in three different formats?

@MaxHillebrand I just merged this PR so that you can take a look at the [preliminary doc site](https://wasabi.incodewetrust.com/GettingStarted.html).

**Comment by thunderBiscuit** [comment]
I just wanted to make sure I wasn't pushing my preferred choice on other people... For now the `config.toml` version is the one used to build the website, but this way the few contributors can have an early discussion on which format to go forward with.
